controllers/operator/enoexecevent_objects.go

```diff
@@ -0,0 +1,216 @@
+			Resources: []string{"pods", "pods/status"},
+			Verbs:     []string{UPDATE},
+		},
+	})
+}
+
+func buildRoleENoExecEventController() *rbacv1.Role {
+	return &rbacv1.Role{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      utils.EnoexecControllerName,
+			Namespace: utils.Namespace(),
+		},
+		Rules: []rbacv1.PolicyRule{
+			{
+				APIGroups: []string{v1beta1.GroupVersion.Group},
+				Resources: []string{v1beta1.ENoExecEventResource},
+				Verbs:     []string{LIST, WATCH, GET, UPDATE, PATCH, CREATE, DELETE},
+			},
+			{
+				APIGroups: []string{""},
+				Resources: []string{"configmaps"},
+				Verbs:     []string{LIST, WATCH, GET, UPDATE, PATCH, CREATE, DELETE},
+			},
+			{
+				APIGroups: []string{"coordination.k8s.io"},
+				Resources: []string{"leases"},
+				Verbs:     []string{LIST, WATCH, GET, UPDATE, PATCH, CREATE, DELETE},
+			},
+		},
+	}
+}
+
+func buildClusterRoleENoExecEventsDaemonSet() *rbacv1.ClusterRole {
+	return buildClusterRole(utils.EnoexecDaemonSet, []rbacv1.PolicyRule{
+		{
+			APIGroups:     []string{"security.openshift.io"},
+			Resources:     []string{"securitycontextconstraints"},
+			ResourceNames: []string{"privileged"},
+			Verbs:         []string{USE},
+		},
+	})
+}
+
+func buildRoleENoExecEventDaemonSet() *rbacv1.Role {
+	return &rbacv1.Role{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      utils.EnoexecDaemonSet,
+			Namespace: utils.Namespace(),
+		},
+		Rules: []rbacv1.PolicyRule{
+			{
+				APIGroups: []string{v1beta1.GroupVersion.Group},
+				Resources: []string{v1beta1.ENoExecEventResource},
+				Verbs:     []string{GET, CREATE},
+			},
+			{
+				APIGroups: []string{v1beta1.GroupVersion.Group},
+				Resources: []string{v1beta1.ENoExecEventResource + "/status"},
+				Verbs:     []string{UPDATE},
+			},
+		},
+	}
+}
+
+// buildDaemonSet returns the DaemonSet object for ENoExecEvent
+func buildDaemonSetENoExecEvent(serviceAccount string, name string, logVerbosity int, args ...string) *appsv1.DaemonSet {
+	return &appsv1.DaemonSet{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      utils.EnoexecDaemonSet,
+			Namespace: utils.Namespace(),
+			Labels: map[string]string{
+				"app": utils.EnoexecDaemonSet,
+			},
+		},
+		Spec: appsv1.DaemonSetSpec{
+			Selector: &metav1.LabelSelector{
+				MatchLabels: map[string]string{
+					"app": utils.EnoexecDaemonSet,
+				},
+			},
+			Template: corev1.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{
+						"app": utils.EnoexecDaemonSet,
+					},
+				},
+				Spec: corev1.PodSpec{
+					ServiceAccountName:           serviceAccount,
+					AutomountServiceAccountToken: utils.NewPtr(true),
+					HostPID:                      true,
+					Containers: []corev1.Container{
+						{
+							Name:            name,
+							Image:           utils.Image(),
+							ImagePullPolicy: corev1.PullIfNotPresent,
+
+							Args: args,
+							Command: []string{
+								"/enoexec-daemon",
+								fmt.Sprintf("--initial-log-level=%d",
```

Comment:
```suggestion
								fmt.Sprintf("--initial-log-level=%d",
```

just to confirm, is the double dash `--` correctly handled?